src/components/ui/dashboard/add-entities.tsx

```diff
@@ -28,27 +25,16 @@ export default function AddEntities() {
         Add New
       </h3>
       <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
-        <CoachingSessionDialog
-          mode="create"
-          open={open}
-          onOpenChange={setOpen}
-          onCoachingSessionUpdated={onCoachingSessionAdded}
-          dialogTrigger={
-            <AddCoachingSessionButton
-              disabled={!isCoach || !currentOrganizationId}
-            />
-          }
+        <AddCoachingSessionButton
+          disabled={!isCoach || !currentOrganizationId}
+          onClick={onCreateSession}
```

Comment:
Yes!! This fulfills my TODO item, so thank you for refactoring this to match the style of AddMemberButton. What a pleasant surprise to discover. :)